deployment/kubernetes/values/secrets-staging.enc.yaml

```diff
@@ -1,21 +1,22 @@
-ethstatsWebsocketSecret: ENC[AES256_GCM,data:hX0n9epVnN+ts9qx6CaJsA==,iv:vl1ph6LEbr98KEgJw737UpJIvxqa6twCmszC+E69paE=,tag:DyEV1OMSVc/F645SnK4t9w==,type:str]
-#ENC[AES256_GCM,data:/WZvrKD3OK7eifPTmhQ83JKVYfpunQoicEmdCA==,iv:8gIedxLC8BuqMu+c8OWPvxctXYGtkpxrdKkemjzNIf8=,tag:d//D722mM8eOpgvHwPCP0w==,type:comment]
-faucetMnemonic: ENC[AES256_GCM,data:X4aDLPLUNGHG+zKuxms3rvET4ySVzD8wkfWsRbPC8fsx5cS06MtRHV6iXJSIV9luNH54fFdrmkSQZFoA4i7Oy4lSXL3Rq2ULllnk9Q==,iv:zhnVcmvcdHt2mRwF6Xv4mqXN5NqgnB1dwubf+jFEH2E=,tag:FQEQ0yTnujBhaNpTRGHz5w==,type:str]
-#ENC[AES256_GCM,data:fLAlcGJpkrLmeumALacf4lekgJESL95GRC1hiWODtuME,iv:60YaVHC0YGe1f7oryyE3+ySatpDE83potZuUtHKwLIg=,tag:DFR37u2R+EK6KlynlvFb6A==,type:comment]
-databaseUsername: ENC[AES256_GCM,data:j09ehw23,iv:/zsi8Nlrc4dEOvOnqbwKVetRex/HQoIt0RYhT74TwJk=,tag:2/vzJJ+CyEaEsef7Ygrsjw==,type:str]
-databasePassword: ENC[AES256_GCM,data:x2y5GHFUeqJfCRfs4Gdcew==,iv:mhrQporXX+E3k3QjKm1DnJJtlPDsA4KaqCkT4IW0XiY=,tag:e+AYGw6lE+7mbu/Yf4EVGQ==,type:str]
-#ENC[AES256_GCM,data:mO3uMF8iLHTIEgKEc2sIXLT3ltjLbaBbGl6/8f/tug==,iv:rSNMY0CyCwwNg/YxqKh84dtuAAi9AbGdoDe58Qy8I9A=,tag:GUSOL/FRyvDsYXX8DYaV/g==,type:comment]
-infuraAccessToken: ENC[AES256_GCM,data:ulWq+Xd+VtAgiM8DbtDHPMiQLso=,iv:DwDr1NF4MQE3fsHJZpqXQfPaWz3Y/vMQoMuXxXVv2XU=,tag:7v0243z+Y6I6uTkOhq4DNw==,type:str]
+bridgeEnvKey: ENC[AES256_GCM,data:GCGBzZKbAYo6Hzm3+n53z0gsL+scqTEw5nFbj/8cK/SplefjMg==,iv:c0fWu2ch5nKpPz3i0oCXNEzN0h4CueUuourSJ6Z1e7k=,tag:W6nxD6frROc4kOsEJEyfCw==,type:str]
+#ENC[AES256_GCM,data:yDtgDiCcJRzu+qV3f0qnV8/wjOMceVf2hfFakPqEbT5G+mR1eZAUqdtn59hk+FAhFSr2sh4c1Y9q,iv:VYTcLDAMreo4JPjgRhNeNyZ6jVTJQVa/5JDXX0+LL6Y=,tag:MchzDW36uftLh8/HwseBfg==,type:comment]
+ethstatsWebsocketSecret: ENC[AES256_GCM,data:uhXMX44PVoe3ASntXFsN7w==,iv:q+UGV/YepgqRcH7csIhbPNgIahaZtBZl4cbGl3/S/Zw=,tag:zppLDtNE1jMo3Ki3g8CS2g==,type:str]
+#ENC[AES256_GCM,data:Y6KnqQipQPp1ObhYwrNHzUnRVsdUljxVVlQ5Lg==,iv:4zfgvGwvdxhUxlWrp+qonwy2J3/JATk49V2Al6An3O4=,tag:4Rpw2mS1u23BGVID8Dp50w==,type:comment]
+faucetMnemonic: ENC[AES256_GCM,data:p+qsH0jgAKYS+d1xJZlcAmjS+E6Mufgy7ElFVac4C6c6k5ukWXES9KRyiyE4tW7SZFV6wrsvYYx3dyePtNeBdtvaeUVSbDV397qIlA==,iv:ayvoanPIwi0XCE3cD3sOmzeyTnfiCOWhZJXUtJvJEpE=,tag:m/eubN81R4+gDZt047TDUA==,type:str]
+#ENC[AES256_GCM,data:xdgI8BE/UIV18Hy9L0y7gVvxu0R9IBa3wjZkX4ufnpn4,iv:iJ63ScvXMCgvWCFvdbgLEQ/KWD8wTvyJfE5qbAl7wqk=,tag:YxFjrQp3qpwh81oFrIHinw==,type:comment]
+databaseUsername: ENC[AES256_GCM,data:JvJbAHwS,iv:XjQNHRxKADxM0gzpHKiH5bwcdwxFGZ9e7TQqM712rzY=,tag:f+7zy7CP/anThHc7mq+96g==,type:str]
+databasePassword: ENC[AES256_GCM,data:FrjulFmgKIzXNvdvFY6N9w==,iv:wd/A2CLv0OD0Ff3J8mH+eUXm1svB2MpvMLS77kurlig=,tag:IJuCTIpomgsTng1K3PQZpQ==,type:str]
+#ENC[AES256_GCM,data:gR2dgKGS7qPtOLUMlkrP9g5Cp4GAQIq7g4aLmMwHRg==,iv:f+GcRzK3x/i1I4yFeuNZUmlde7KLh9U9qn5SiFVUHas=,tag:/3vDQF4m5aTKyYHN1meUpA==,type:comment]
+infuraAccessToken: ENC[AES256_GCM,data:4UlrgV5HZcCryafGeBJYmbOytPg=,iv:0dEfonb98hBA0RHxDBlF9W9fgWBCTi6xUJ6L07g+UdU=,tag:abQm8hf8JihpW/9HEycp/Q==,type:str]
```

Comment:
Is all of this information supposed to be public?

Reply:
The values are encrypted so I don't think it is a problem ?